### PR TITLE
explore optimized es search queries

### PR DESCRIPTION
**Pull request by szieglerICF**
Test boolean query with varying boost values to optimize search results

### CODE REVIEW

File: etl/load_es/load_es.py [hunk 1]
```diff
@@ -40,10 +42,10 @@ def main():
 				count = count + 1
 				if count == 1:
 					field_names = line.split("|")
-					assert(len(field_names) == 10, "Expected 10 piped fields")
+					assert(len(field_names) = 10, "Expected 10 piped fields")
```
Comment (rppatel1007) [inline etl/load_es/load_es.py]: Since this is an assert, it should stay '=='

File: etl/load_es/load_es.py [hunk 2]
```diff
@@ -40,10 +42,10 @@ def main():
...
 				else:
 					fields = line.strip().split("|")
-					assert(len(fields) == 10, "Expected 10 piped fields")
+					assert(len(fields) = 10, "Expected 10 piped fields")
```
Comment (rppatel1007) [inline etl/load_es/load_es.py]: Since this is an assert, it should stay '=='

File: etl/load_es/load_es.py (hunk 1 above)
Comment (szieglerICF) [reply etl/load_es/load_es.py]: Corrected the syntax. I tested the code changing the assert value to 9 to see the assert properly trigger. Reset to correct value of 10.

Comment (rppatel1007) [reply etl/load_es/load_es.py]: 👍

**Pull request merged**